u8timeseries/utils/statistics.py

```diff
@@ -0,0 +1,204 @@
+from ..timeseries import TimeSeries
```

Comment:
Is this a brand new file or was it moved from somewhere?

Reply:
This was moved from the the models module. Julien suggested that the models module should only contain forecasting (and probably regression) models. @hrzn please correct me if I'm wrong here.